tests/unit/ValidationFactory.spec.js

```diff
@@ -0,0 +1,47 @@
label describes the button\'s text'}},{'type':'FormInput','field':'name','config':{'name':'Key Name','label':'Key Name','helper':'A variable key name is a symbolic name to reference information.','validation':'regex:/^(?:[A-Z_a-z])(?:[0-9A-Z_a-z])*$/|required'}},{'type':'FormInput','field':'fieldValue','config':{'label':'Value','helper':'The value being submitted'}},{'type':'FormMultiselect','field':'variant','config':{'label':'Button Variant Style','helper':'The variant determines the appearance of the button','options':[{'value':'primary','content':'Primary'},{'value':'secondary','content':'Secondary'},{'value':'success','content':'Success'},{'value':'danger','content':'Danger'},{'value':'warning','content':'Warning'},{'value':'info','content':'Info'},{'value':'light','content':'Light'},{'value':'dark','content':'Dark'},{'value':'link','content':'Link'}]}},{'type':'FormInput','field':'conditionalHide','config':{'label':'Visibility Rule','helper':'This control is hidden until this expression is true'}},{'type':'FormInput','field':'customCssSelector','config':{'label':'CSS Selector Name','helper':'Use this in your custom css rules','validation':'regex: [-?[_a-zA-Z]+[_-a-zA-Z0-9]*]'}}],'editor-control':'FormButton','editor-component':'FormButton'}]],'label':'Multicolumn / Table','config':{'icon':'fas fa-table','label':null,'options':[{'value':1,'content':'4'},{'value':2,'content':'4'},{'value':3,'content':'4'}]},'component':'FormMultiColumn','container':true,'inspector':[{'type':'ContainerColumns','field':'options','config':{'label':'Column Width','helper':null}},{'type':'ColorSelect','field':'color','config':{'label':'Text Color','helper':'Set the element\'s text color','options':[{'value':'text-primary','content':'primary'},{'value':'text-secondary','content':'secondary'},{'value':'text-success','content':'success'},{'value':'text-danger','content':'danger'},{'value':'text-warning','content':'warning'},{'value':'text-info','content':'info'},{'value':'text-light','content':'light'},{'value':'text-dark','content':'dark'}]}},{'type':'ColorSelect','field':'bgcolor','config':{'label':'Background Color','helper':'Set the element\'s background color','options':[{'value':'alert alert-primary','content':'primary'},{'value':'alert alert-secondary','content':'secondary'},{'value':'alert alert-success','content':'success'},{'value':'alert alert-danger','content':'danger'},{'value':'alert alert-warning','content':'warning'},{'value':'alert alert-info','content':'info'},{'value':'alert alert-light','content':'light'},{'value':'alert alert-dark','content':'dark'}]}},{'type':'FormInput','field':'conditionalHide','config':{'label':'Visibility Rule','helper':'This control is hidden until this expression is true'}},{'type':'FormInput','field':'customCssSelector','config':{'label':'CSS Selector Name','helper':'Use this in your custom css rules','validation':'regex: [-?[_a-zA-Z]+[_-a-zA-Z0-9]*]'}}],'editor-control':'MultiColumn','editor-component':'MultiColumn'}]}];
```

Comment:
I think this _really_ long line would be better in a separate file, like `testConfig.json`.

Reply:
I added the file with screen config